pkg/client/openstack/admin/client.go

```diff
@@ -23,25 +26,31 @@ type AdminClient interface {
 	DeleteUser(username, domainName string) error
 	GetKubernikusCatalogEntry() (string, error)
 	GetRegion() (string, error)
+	CreateContainer(containerName, serviceUserName, serviceUserDomainName, serviceUserProjectID string) error
```

Comment:
Maybe we should call this method `CreateStorageContainer` to make it clearer which type of container.